runtime.md

```diff
@@ -11,8 +11,8 @@ The state of a container includes the following properties:
 
 * **`ociVersion`** (string, REQUIRED) is the OCI specification version used when creating the container.
 * **`id`** (string, REQUIRED) is the container's ID.
-This MUST be unique across all containers on this host.
-There is no requirement that it be unique across hosts.
+    This MUST be unique across all containers on this host.
+    There is no requirement that it be unique across hosts.
 * **`status`** (string, REQUIRED) is the runtime state of the container.
```

Comment:
@TomSweeneyRedHat  I'm not sure what's your browser. This may be that's the browser's problem. If you use another browser to view, does it look OK?